addons/html_editor/static/src/main/table/table_ui_plugin.js

```diff
@@ -42,11 +42,7 @@ export class TableUIPlugin extends Plugin {
                     if (this.config.direction === "rtl") {
                         // position from the right instead of the left as it is needed
                         // to ensure the expand animation is properly done
```

Comment:
Does this comment still apply? And are we sure the expand animation is properly done after your change?

Reply:
Indeed, the comment is about the expand animation should go to the left in `rtl` language, only the computation of the right position was wrong